### ISSUE TITLE
Literature Review Pre-submission 1

### ISSUE THREAD

**Issue by sedurCode** [sedurCode/PhD#1]
The first submission of my lit review is due on the 26th of April to Damian. 
There is no way I am going to make it, but we will strive and learn from the organisation lessons (hence the filling of this repo with issues and projects). 

In order to have something to show to Damian and Duncan, I need to get a basic document with some research streams together for the date above. This will include the following: 

- [x] The creation of a project template
- [x] The creation of the sections for the lit review
- [ ] The aggregation of literature
- [ ] The creation of research questions
- [ ] The writing of background discussion on the latest in the appropriate fields.

What I have failed to do is to create good notes on literature in order to create summaries. I have also spent too much time thinking about some particular stuff (legislation etc) and not about others such as soundscape, brain caps, the problems with L_den and how we could make a new perceptually motivated environmental noise metric.  

These are all lessons learned, and I need to work towards remedying them in the future. 
Approaches to these include:

- organising and tracking the workflow for literature review
- form a quasi-systematic literature review process
- Identify a good method of background reading - reading and evaluating the right kind of literature in order to identify fields of further investigation
- performing a systematic process for further issue and solution identification in the further reading of subtopics
- identifying research gaps in literature, as well as themes of though and evolutions of methods and practice etc
- Skim reading and identification of appropriateness from abstracts
- Correctly labelling and organising literature, including summaries that can be used in the review
- Taking note of trends and connections in literature (in the journal)
- Taking good notes and using the journal as a journal to mark the research journey

Basically, write more useful stuff! 
Be prolific and systematic, and this will help you publish more and do more good stuff!
